components/glow.lua

```diff
@@ -371,5 +372,38 @@ binder:SetScript("OnEvent", function()
         end
     end
 
+    -- Support for AzeriteUI5's LibActionButton
+    if (LAB_GE) then
+        local buttonUpdateFunc = function(lib, event, self)
+            if (self._state_type ~= "action") then
+                -- If button is not an "action", then GetSpellId is unusable
+                -- This happens for instance with vehicle buttons
+                -- They are probably not meant to glow, so it's simpler to just ignore them
+                return
+            end
+            if (not self.GetGlowID) then
+                self.GetGlowID = self.GetSpellId;
+            end
+            if (not self.EnableGlow) then
+                self.EnableGlow = function(button)
+                    button:SetSpellActivationColor(1, 1, 1);
+                    button:ShowSpellActivation();
+                end
+            end
+            if (not self.DisableGlow) then
+                self.DisableGlow = function(button)
+                    button:HideSpellActivation();
+                end
+            end
+            SAO:UpdateActionButton(self);
+        end
+
+        local LAB_GEButtonUpdateFunc = function(event, self)
+            buttonUpdateFunc(LAB_GE, event, self);

```

Comment:
`LAB_GE` is passed as argument, but never used in practice. Please read first review comment for suggestion to improve this.

Reply:
As mentioned above, in my latest commit I embedded the glow methods into LAB_GE instead, so now it makes sense to pass LAB_GE as an argument again. 

The latest commit now only checks for the library instance, defines LAB_GEButtonUpdateFunc, and registers it as a callback in LAB_GE if appropriate.